Makefile

```diff
@@ -1,8 +1,18 @@
 IMAGE_NAME=quay.io/theforeman/candlepin
-IMAGE_TAG=4.4.14
+IMAGE_TAG=4.4.20-1
+
+PROJECT_XY_TAG=4.4
+PROJECT_XYZ_TAG=4.4.20
+
+FOREMAN_XY_TAG=3.15
+FOREMAN_XYZ_TAG=3.15.0
```

Comment:
We'll end up managing these versions across multiple of these repositories and I wonder how to properly keep this in sync. This could perhaps instead be handled at the CI level rather than here? 

@ekohl Could your ideas around shared Github Actions help?

Reply:
> Then I also wonder about putting it into files vs using git tags. Though changing a tag will be a really bad idea, it could be easy for visibility that it's really the same thing.

And would we git tag all the container tags for parity?

Reply:
That was my thinking, but I don't know how feasible it really is. Perhaps foreman-x.y should really be a branch and foreman-x.y.z a tag. Then we'd need x.y.z-a for bugfixes though.

Perhaps we leave out x.y.z altogether?